tests/update_only_fields/tests.py

```diff
@@ -256,6 +257,29 @@ def post_save_receiver(**kwargs):
         pre_save.disconnect(pre_save_receiver)
         post_save.disconnect(post_save_receiver)
 
+    def test_empty_update_fields_positional_save(self):
+        s = Person.objects.create(name="Sara", gender="F")
+
+        msg = "Passing positional arguments to save() is deprecated"
+        with (
+            self.assertWarnsMessage(RemovedInDjango60Warning, msg),
+            self.assertNumQueries(0),
+        ):
+            s.save(False, False, None, [])
+
+    async def test_empty_update_fields_positional_asave(self):
+        s = await Person.objects.acreate(name="Sara", gender="F")
+        # Workaround for a lack of async assertNumQueries.
+        s.name = "Other"
+
+        msg = "Passing positional arguments to asave() is deprecated"
+        with (self.assertWarnsMessage(RemovedInDjango60Warning, msg),):
```

Comment:
This is almost not worth mentioning, but would you prefer to un-parenthesize this or was this intentional?

Reply:
Thanks for raising this! Is a leftover when I was fighting assertNumQueries not working with async. Fixing ASAP! :bow: